2024-05-26 14:38:36,137 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-26 14:38:36,137 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Configure stats pid to 138624
2024-05-26 14:38:36,137 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Loading settings from /home/inaki/.config/wandb/settings
2024-05-26 14:38:36,137 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Loading settings from /home/inaki/host_data/wandb/settings
2024-05-26 14:38:36,137 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-26 14:38:36,137 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-26 14:38:36,137 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_init.py:_log_setup():520] Logging user logs to /home/inaki/host_data/wandb/run-20240526_143836-3t34xkbz/logs/debug.log
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_init.py:_log_setup():521] Logging internal logs to /home/inaki/host_data/wandb/run-20240526_143836-3t34xkbz/logs/debug-internal.log
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_init.py:_jupyter_setup():466] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9880cfbc40>
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_init.py:init():560] calling init triggers
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {'MAX_LENGTH': 512, 'HEAD_DROPOUT': 0.0, 'DATA_AUGMENTATION': ['WR'], 'FOLDS_NUM': 5, 'NUM_EPOCHS': 5, 'LEARNING_RATE': 2e-05, 'BATCH_SIZE': 16, 'FOLD_IDX': 0, 'LOSS_FN': 'cross_entropy', 'SEED': 5581}
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_init.py:init():610] starting backend
2024-05-26 14:38:36,138 INFO    MainThread:138624 [wandb_init.py:init():614] setting up manager
2024-05-26 14:38:36,140 INFO    MainThread:138624 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-05-26 14:38:36,142 INFO    MainThread:138624 [wandb_init.py:init():622] backend started and connected
2024-05-26 14:38:36,151 INFO    MainThread:138624 [wandb_run.py:_label_probe_notebook():1328] probe notebook
2024-05-26 14:38:36,152 INFO    MainThread:138624 [wandb_run.py:_label_probe_notebook():1338] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-26 14:38:36,152 INFO    MainThread:138624 [wandb_init.py:init():711] updated telemetry
2024-05-26 14:38:36,153 INFO    MainThread:138624 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-26 14:38:36,531 INFO    MainThread:138624 [wandb_run.py:_on_init():2396] communicating current version
2024-05-26 14:38:36,580 INFO    MainThread:138624 [wandb_run.py:_on_init():2405] got version response 
2024-05-26 14:38:36,580 INFO    MainThread:138624 [wandb_init.py:init():795] starting run threads in backend
2024-05-26 14:38:36,908 INFO    MainThread:138624 [wandb_run.py:_console_start():2374] atexit reg
2024-05-26 14:38:36,908 INFO    MainThread:138624 [wandb_run.py:_redirect():2229] redirect: wrap_raw
2024-05-26 14:38:36,909 INFO    MainThread:138624 [wandb_run.py:_redirect():2294] Wrapping output streams.
2024-05-26 14:38:36,909 INFO    MainThread:138624 [wandb_run.py:_redirect():2319] Redirects installed.
2024-05-26 14:38:36,910 INFO    MainThread:138624 [wandb_init.py:init():838] run started, returning control to user process
2024-05-26 14:39:10,289 INFO    MainThread:138624 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-26 14:39:10,290 INFO    MainThread:138624 [wandb_init.py:_pause_backend():431] pausing backend
